src/main/java/seedu/address/model/book/HideAllBooks.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.model.book;
+
+import java.util.function.Predicate;
+
+/**
+ * Set a predicate that all books are not available when encrypting
+ */
+public class HideAllBooks implements Predicate<Book> {
```

Comment:
What in the world is this class for. (especially since your test just returns false?)